### PR TITLE
Researchers

### PR DESCRIPTION
**Pull request by LetsTrie**
### Completed API

1. As an audience, I would like to receive thank you message from Researcher in my mail after completing the survey

2. As an Admin, I can view all kinds of data

3. As an Admin, I can view 1st 200 users, they will be selected for rewards

4. As an Admin, I would like to send a promotional email to all users.

5. As an Admin, I would like to see the list of disapproved users

### CODE REVIEW

File: server/researcher/researcher.controllers.js [hunk 1]
```diff
@@ -6,7 +6,6 @@ module.exports.submitQuestions = (req, res) => {
   if (userAnswerType.length !== numberOfQuestion) {
 
   }
-  console.log(userAnswerType, correctAnswerType);
   for (let i = 0; i < userAnswerType.length; i++) {
     console.log()
```
Comment (orvi2014) [inline server/researcher/researcher.controllers.js]: unused console.log here

File: server/researcher/researcher.controller.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
+const { numberOfQuestion, correctAnswerType, userAnswerType, vaildData } = require('./researcher.dummyData');
```
Comment (orvi2014) [inline server/researcher/researcher.controller.js]: this line is no longer.

File: index.routes.js [hunk 3]
```diff
@@ -1,12 +1,27 @@
+<<<<<<< HEAD
```
Comment (orvi2014) [inline index.routes.js]: merge conflict here

File: server/account/account.controller.js [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+      return res
+        .status(400)
+        .json({ success: false, message: 'Phone Number already Exists' });
+=======
+    if (user) return res.status(400).json({ message: "Email already Exists" });
+    // console.log(data, user);
+    user = await AccountModel.findOne({ phoneNumber: data.phoneNumber });
+    if (user)
+      return res.status(400).json({ message: "Phone Number already Exists" });
+>>>>>>> 68fd295c3d6fea4ff9c5092fee14d43e37484503
```
Comment (orvi2014) [inline server/account/account.controller.js]: merge conflict is here

File: server/account/account.controller.js [hunk 5]
```diff
@@ -0,0 +1,129 @@
...
+    const newUser = new AccountModel({
+      name: data.name,
+      email: data.email,
+      phoneNumber: data.phoneNumber,
+      role: data.role,
+      password: hashedPassword
+    });
+    await newUser.save();
+    newUser.password = undefined;
+<<<<<<< HEAD
```
Comment (orvi2014) [inline server/account/account.controller.js]: merge conflict

File: server/admin/admin.controller.js [hunk 6]
```diff
@@ -0,0 +1,170 @@
...
+const colors = require('colors');
+
+const { AccountModel } = require('../account/account.model');
+
+module.exports.allQuestionsSet = async (req, res) => {
+  try {
+    const questionList = await ResearcherQuestionSetModel.find();
+    return res.status(200).json({ success: true, questionList });
+  } catch (error) {
+    return res.status(500).json({
```
Comment (orvi2014) [inline server/admin/admin.controller.js]: It should be pass into next middleware.

File: config/express.js [hunk 7]
```diff
@@ -1,22 +1,22 @@
-const express = require('express');
-const bodyParser = require('body-parser');
-const logger = require('morgan');
-const cookieParser = require('cookie-parser');
-const compress = require('compression');
-const helmet = require('helmet');
-const passport = require('passport');
-const config = require('./config');
-const routes = require('../index.routes');
+const express = require("express");
```
Comment (orvi2014) [inline config/express.js]: Use single quote everywhere.

File: server/researcher/researcher.controller.js [hunk 8]
```diff
@@ -1,80 +1,161 @@
...
+    const review = await AudienceQuestionSubmitModel.find({
+      researcherID,
+      QuestionSetID
+    });
     return res.status(200).json({
       success: true,
-      accessToken,
-      user: vaildData[getIndex]
-    })
+MONGODB_HOST = mongodb+srv://oabd:oabd12345@feedback-db-8sbpq.mongodb.net/feedback?retryWrites=true&w=majority
```
Comment (orvi2014) [inline server/researcher/researcher.controller.js]: Remove this line

File: server/researcher/researcher.controllers.js (hunk 1 above)
Comment (LetsTrie) [reply server/researcher/researcher.controllers.js]: DONE

File: server/researcher/researcher.controller.js (hunk 2 above)
Comment (LetsTrie) [reply server/researcher/researcher.controller.js]: DONE

File: index.routes.js (hunk 3 above)
Comment (LetsTrie) [reply index.routes.js]: DONE

File: server/account/account.controller.js (hunk 4 above)
Comment (LetsTrie) [reply server/account/account.controller.js]: DONE

File: server/account/account.controller.js (hunk 5 above)
Comment (LetsTrie) [reply server/account/account.controller.js]: DONE

**Pull request merged**